### PR TITLE
Updated Requirements-windows and fetch real time from chatbot.

### PR DESCRIPTION
**Pull request by Vishi13**
Changes the requirements-windows.txt as some libraries were missing and the versions of libraries written earlier were not compatible with the python version, so i have removed the versions so that now libararies got automatically with compatibility.
Also, earlier, when we ask time from chatbot, it gives the same time , when we started running the chatbot. Now, changes are made to display the exact time.

### CODE REVIEW

**Comment by satyammittal** [comment]
@Vishi13 why file name update of requirements_windows?

**Comment by Vishi13** [comment]
@satyammittal  Sorry, mistakenly done.
Fixed the filename to requirements_windows.

File: chatbot.py [hunk 1]
```diff
@@ -136,9 +136,10 @@
         engine.say(w.get_temperature('celsius'))
         engine.runAndWait()
     elif translate in var3:
+        n1 = datetime.datetime.now()
```
Comment (satyammittal) [inline chatbot.py]: please use better variable name. like current_time

Comment (Vishi13) [reply chatbot.py]: Done! Updated the code.

File: requirements_windows.txt [hunk 2]
```diff
@@ -1,82 +1,86 @@
...
-wcwidth==0.1.8
-webencodings==0.5.1
-win-inet-pton==1.1.0
-wincertstore==0.2
-xmltodict==0.12.0
-zipp==3.1.0
-pyjokes==0.6.0
-(base)
+anaconda-client
+#anaconda-navigator
```
Comment (satyammittal) [inline requirements_windows.txt]: Why this is commented?

Comment (Vishi13) [reply requirements_windows.txt]: Actually, I don't use anaconda. So, Installing anaconda-navigator is giving error and some of other installation related to anaconda are giving error and i have commented them. 
Should I remove the comments or resolve it or leave as it is?

Comment (satyammittal) [reply requirements_windows.txt]: leave it for now, you can add that in separate pr to resolve the issue

Comment (Vishi13) [reply requirements_windows.txt]: Done! Please check it.

File: google_places.py [hunk 3]
```diff
@@ -61,8 +61,8 @@ def change_location_query(address, key):
         address, key)
     r = requests.get(url)
     x = r.json()
+    print(x)
```
Comment (satyammittal) [inline google_places.py]: why this change?

Comment (Vishi13) [reply google_places.py]: I was checking the response returned by API as it was giving error that you have to add the billing account to continue using it. Should I remove it?

Comment (satyammittal) [reply google_places.py]: yes, remove it

File: chatbot.py [hunk 4]
```diff
@@ -135,11 +135,16 @@
         engine.runAndWait()
         engine.say(w.get_temperature('celsius'))
         engine.runAndWait()
-    elif translate in var3:
-        print("Current date and time : ")
-        print(now.strftime("The time is %H:%M"))
-        engine.say(now.strftime("The time is %H:%M"))
-        engine.runAndWait()
+    elif (translate in var3 or translate in var5):
```
Comment (satyammittal) [inline chatbot.py]: no need to use () parentheses

Comment (Vishi13) [reply chatbot.py]: Removed the parentheses

File: google_places.py (hunk 3 above)
Comment (Vishi13) [reply google_places.py]: Removed the comment also.
